key/keys.go

```diff
@@ -133,10 +152,15 @@ func (p *Pair) FromTOML(i interface{}) error {
 	if !ok {
 		return errors.New("private can't decode toml from non PairTOML struct")
 	}
-
+	scheme := crypto.SchemeFromName(ptoml.SchemeName)
+	if scheme == nil {
+		return fmt.Errorf("invalid Scheme name in Pair FromTOML: %s", ptoml.SchemeName)
```

Comment:
setting the default scheme instead might be an option to handle old TOML stuff, I guess.